StudentDetails.java

```diff
@@ -0,0 +1,155 @@
+import java.util.*;
+import java.util.Scanner;
+
+class StudentData {
+    private int sno;
+    private String sfname;
+    private String slname;
+
+    StudentData(int sno, String sfname, String slname) {
+        this.sno = sno;
+        this.sfname = sfname;
+        this.slname = slname;
+    }
+
+    public int getSno() {
+        return sno;
+    }
+
+    public String getSlname() {
+        return slname;
+    }
+
+    public String getSfname() {
+        return sfname;
+    }
+
+    public String toString() {
+        return sno + " " + sfname + " " + slname;
+    }
+}
+
+class StudentDetails {
+    private static final int ADD_STUDENT = 1;
+    private static final int SHOW_STUDENT = 2;
+    private static final int SEARCH_STUDENT = 3;
+    private static final int REMOVE_STUDENT = 4;
+    private static final int UPDATE_STUDENT = 5;
+
+    public static void main(String[] args) {
+
+        List<StudentData> c = new ArrayList<StudentData>();
+        Scanner obj = new Scanner(System.in);
```

Comment:
Use appropriate names! Every Java programmer knows, that if you use the __new__ keyword you instantiate a new object. So the name __obj__ adds no meaning at all. __obj__ can be anything. A object of __StudentDetails__, a object of __StudentData__, a object of __Scanner__, a object of whatever. Just use __scanner__ or __inputReader__, or something that gives the user a hint what that variable is or it's purpose.